tool/tctl/common/auth_command.go

```diff
@@ -120,6 +121,7 @@ func (a *AuthCommand) Initialize(app *kingpin.Application, config *service.Confi
 	a.authSign.Flag("leaf-cluster", `Leaf cluster to generate identity file for when --format is set to "kubernetes"`).StringVar(&a.leafCluster)
 	a.authSign.Flag("kube-cluster-name", `Kubernetes cluster to generate identity file for when --format is set to "kubernetes"`).StringVar(&a.kubeCluster)
 	a.authSign.Flag("app-name", `Application to generate identity file for`).StringVar(&a.appName)
+	a.authSign.Flag("db-name", `Database to generate identity file for`).StringVar(&a.dbName)
```

Comment:
I assume `--app-name` and `--db-name` are mutually exclusive here?

We should probably check for cases when both are specified and print a nice error rather than allowing app to silently take precedence.